plugins/woocommerce/src/Internal/Admin/Settings/Payments.php

```diff
@@ -175,6 +175,39 @@ function ( $a, $b ) {
 		return $payment_providers;
 	}
 
+	/**
+	 * Get the business location country code for the Payments settings.
+	 *
+	 * @return string The ISO 3166-1 alpha-2 country code to use for the overall business location.
+	 *                If the user didn't set a location, the WC base location country code is used.
+	 */
+	public function get_country(): string {
+		$user_nox_meta = get_user_meta( get_current_user_id(), self::USER_PAYMENTS_NOX_PROFILE_KEY, true );
+		if ( ! empty( $user_nox_meta['business_country_code'] ) ) {
+			return $user_nox_meta['business_country_code'];
+		}
+
+		return WC()->countries->get_base_country();
+	}
+
+	/**
+	 * Set the business location country for the Payments settings.
+	 *
+	 * @param string $location The country code. This should be a ISO 3166-1 alpha-2 country code.
+	 */
+	public function set_country( string $location ): bool {
+		$user_payments_nox_profile = get_user_meta( get_current_user_id(), self::USER_PAYMENTS_NOX_PROFILE_KEY, true );
+
+		if ( empty( $user_payments_nox_profile ) ) {
+			$user_payments_nox_profile = array();
+		} else {
+			$user_payments_nox_profile = maybe_unserialize( $user_payments_nox_profile );
+		}
+		$user_payments_nox_profile['business_country_code'] = $location;
+
+		return update_user_meta( get_current_user_id(), self::USER_PAYMENTS_NOX_PROFILE_KEY, $user_payments_nox_profile );
```

Comment:
We need to cast it to bool because `update_user_meta` might return the meta ID if first created.

```suggestion
		return false !== update_user_meta( get_current_user_id(), self::USER_PAYMENTS_NOX_PROFILE_KEY, $user_payments_nox_profile );
```